--- Makefile.PL_old	2009-02-20 14:27:05.000000000 +0100
+++ Makefile.PL	2009-02-20 14:27:21.000000000 +0100
@@ -123,23 +123,6 @@
   $prereq_pm->{'File::HomeDir'} = 0.69;
 }
 
-if ($HAVE_FILE_SPEC) {
-  # import PAUSE public key to user's keychain
-  require Config;
-  my $dir;
-  for $dir (split /$Config::Config{path_sep}/, $ENV{PATH}) {
-    my $abs = File::Spec->catfile($dir, 'gpg');
-    my $cmd;
-    if ($cmd = MM->maybe_command($abs)) {
-      print "Importing PAUSE public key into your GnuPG keychain... ";
-      system($cmd, '--quiet', '--import', <PAUSE*.pub>);
-      print "done!\n";
-      print "(You may wish to trust it locally with 'gpg --lsign-key 450F89EC')\n";
-      last;
-    }
-  }
-}
-
 my @sign = (MM->can("signature_target") ? (SIGN => 1) : ());
 # warn "sign[@sign]";
 WriteMakefile(
