diff -rupN a/t/01-app.t b/t/01-app.t
--- a/t/01-app.t        2013-03-07 12:14:42.000000000 +0400
+++ b/t/01-app.t        2014-06-21 18:50:24.316538333 +0400
@@ -5,7 +5,7 @@ use warnings;
 
 use Test::More tests => 3;
 
-use Dancer2 qw(:syntax :tests);
+use Dancer2;
 use Dancer2::Plugin::Paginator;
 
 use t::lib::App;
diff -rupN a/lib/Dancer2/Plugin/Paginator.pm b/lib/Dancer2/Plugin/Paginator.pm
--- a/lib/Dancer2/Plugin/Paginator.pm   2013-03-07 12:14:42.000000000 +0400
+++ b/lib/Dancer2/Plugin/Paginator.pm   2014-06-21 18:57:36.298909865 +0400
@@ -6,7 +6,7 @@ package Dancer2::Plugin::Paginator;
 use strict;
 use warnings;
 
-use Dancer2 ':syntax';
+use Dancer2;
 use Dancer2::Plugin;
 
 use Paginator::Lite;
