From 848a973011d2fc8903b333522c65dfe54acd32d5 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentfredric@gmail.com>
Date: Mon, 31 Jan 2011 20:53:31 +1300
Subject: [PATCH 2/2] Apply part of 1.94-SignatureTest.patch

---
 t/00signature.t |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/t/00signature.t b/t/00signature.t
index fbb9ae2..62667da 100644
--- a/t/00signature.t
+++ b/t/00signature.t
@@ -57,6 +57,9 @@ BEGIN {
             $exit_message = "Cannot connect to the keyserver";
         }
     }
+    unless ($exit_message) {
+        $exit_message = "DIST patched by Gentoo, invalidating SIGNATURE"
+    }
     if ($exit_message) {
         $|=1;
         print "1..0 # SKIP $exit_message\n";
-- 
1.7.6

