--- cflags.SH.orig	2007-07-05 13:33:47.000000000 -0400
+++ cflags.SH	2007-07-05 13:34:15.000000000 -0400
@@ -330,6 +330,8 @@ if test -f .patch; then
 fi
 
     : Can we perhaps use $ansi2knr here
+    [ "x$file" = xregcomp ] && export ccflags="${ccflags} -fno-stack-protector"
+    [ "x$file" = xregexec ] && export ccflags="${ccflags} -fno-stack-protector"
     echo "$cc -c -DPERL_CORE $ccflags $stdflags $optimize $warn $extra"
     eval "$also "'"$cc -DPERL_CORE -c $ccflags $stdflags $optimize $warn $extra"'
 
