--- a/nikto-2.1.3/nikto.pl	2010-09-07 08:55:51.000000000 -0400
+++ b/nikto-2.1.3/nikto.pl	2010-11-08 19:58:10.000000000 -0500
@@ -6,7 +6,7 @@
 use Getopt::Long;
 Getopt::Long::Configure('no_ignore_case');
 
-# use LW2;                   ### Change this line to use a different installed version
+use LW2;                   ### Change this line to use a different installed version
 
 ###############################################################################
 #                               Nikto                                         #
@@ -52,7 +52,7 @@
 $NIKTO{'DIV'}         = "-" x 75;
 $NIKTO{'version'}     = "2.1.3";
 $NIKTO{'name'}        = "Nikto";
-$NIKTO{'configfile'}  = "/etc/nikto.conf";    ### Change this line if it's having trouble finding it
+$NIKTO{'configfile'}  = "/etc/nikto/nikto.conf";    ### Change this line if it's having trouble finding it
 
 # put a signal trap so we can close down reports properly
 $SIG{'INT'} = \&safe_quit;
@@ -89,7 +89,6 @@
 require "$NIKTOCONFIG{'PLUGINDIR'}/nikto_core.plugin";
 nprint("T:" . localtime($COUNTERS{'startsec'}) . ": Starting", "d");
 require "$NIKTOCONFIG{'PLUGINDIR'}/nikto_single.plugin";
-require "$NIKTOCONFIG{'PLUGINDIR'}/LW2.pm";
 
 my ($a, $b) = split(/\./, $LW2::VERSION);
 die("- You must use LW2 2.4 or later\n") if ($a != 2 || $b < 4);
