--- nikto.pl.orig	2010-02-02 14:28:54.000000000 -0500
+++ nikto.pl	2010-02-18 16:20:21.000000000 -0500
@@ -49,7 +49,7 @@
 $NIKTO{'DIV'}        = "-" x 75;
 $NIKTO{'version'}    = "2.1.1";
 $NIKTO{'name'}       = "Nikto";
-$NIKTO{'configfile'} = "/etc/nikto.conf";    ### Change this line if your setup is having trouble finding it
+$NIKTO{'configfile'} = "/etc/nikto/nikto.conf";    ### Change this line if your setup is having trouble finding it
 
 # read the --config option
 {
