--- nikto.pl.orig	2010-02-02 14:28:54.000000000 -0500
+++ nikto.pl	2010-02-22 15:22:19.000000000 -0500
@@ -49,7 +49,7 @@
 $NIKTO{'DIV'}        = "-" x 75;
 $NIKTO{'version'}    = "2.1.1";
 $NIKTO{'name'}       = "Nikto";
-$NIKTO{'configfile'} = "/etc/nikto.conf";    ### Change this line if your setup is having trouble finding it
+$NIKTO{'configfile'} = "/etc/nikto/nikto.conf";    ### Change this line if your setup is having trouble finding it
 
 # read the --config option
 {
@@ -85,9 +85,7 @@
 require "$NIKTOCONFIG{'PLUGINDIR'}/nikto_core.plugin";
 nprint("T:$starttime: Starting", "d");
 require "$NIKTOCONFIG{'PLUGINDIR'}/nikto_single.plugin";
-require "$NIKTOCONFIG{'PLUGINDIR'}/LW2.pm";
-
-# use LW2;                   ### Change this line to use a different installed version
+use LW2;
 
 ($a, $b) = split(/\./, $LW2::VERSION);
 die("- You must use LW2 2.4 or later\n") if ($a != 2 || $b < 4);
