diff --git a/MooseX-StrictConstructor-0.12b/lib/MooseX/StrictConstructor/Role/Meta/Class.pm b/MooseX-StrictConstructor-0.12b/lib/MooseX/StrictConstructor/Role/Meta/Class.pm
new file mode 100644
index 0000000..02d7b52
--- /dev/null
+++ b/MooseX-StrictConstructor-0.12b/lib/MooseX/StrictConstructor/Role/Meta/Class.pm
@@ -0,0 +1,88 @@
+package MooseX::StrictConstructor::Role::Meta::Class;
+BEGIN {
+  $MooseX::StrictConstructor::Role::Meta::Class::VERSION = '0.12';
+}
+
+use strict;
+use warnings;
+
+use B ();
+use Carp ();
+
+use Moose::Role;
+
+around '_inline_BUILDALL' => sub {
+    my $orig = shift;
+    my $self = shift;
+
+    my @source = $self->$orig();
+
+    my @attrs = (
+        '__INSTANCE__ => 1,',
+        map { B::perlstring($_) . ' => 1,' }
+        grep {defined}
+        map  { $_->init_arg() } $self->get_all_attributes()
+    );
+
+    return (
+        @source,
+        'my %attrs = (' . join(' ', @attrs) . ');',
+        'my @bad = sort grep { !$attrs{$_} } keys %{ $params };',
+        'if (@bad) {',
+            'Carp::confess "Found unknown attribute(s) passed to the constructor: @bad";',
+        '}',
+    );
+};
+
+no Moose::Role;
+
+1;
+
+# ABSTRACT: A role to make immutable constructors strict
+
+
+
+=pod
+
+=head1 NAME
+
+MooseX::StrictConstructor::Role::Meta::Method::Constructor - A role to make immutable constructors strict
+
+=head1 VERSION
+
+version 0.12
+
+=head1 SYNOPSIS
+
+  Moose::Util::MetaRole::apply_metaroles(
+      for_class => $caller,
+      class     => {
+          constructor =>
+              ['MooseX::StrictConstructor::Role::Meta::Method::Constructor'],
+      },
+  );
+
+=head1 DESCRIPTION
+
+This role simply wraps C<_inline_BUILDALL()> (from
+C<Moose::Meta::Class>) so that immutable classes have a
+strict constructor.
+
+=head1 AUTHOR
+
+Dave Rolsky <autarch@urth.org>
+
+=head1 COPYRIGHT AND LICENSE
+
+This software is Copyright (c) 2010 by Dave Rolsky.
+
+This is free software, licensed under:
+
+  The Artistic License 2.0
+
+=cut
+
+
+__END__
+
+
diff --git a/MooseX-StrictConstructor-0.12/lib/MooseX/StrictConstructor/Role/Meta/Method/Constructor.pm b/MooseX-StrictConstructor-0.12/lib/MooseX/StrictConstructor/Role/Meta/Method/Constructor.pm
deleted file mode 100644
index 0eed129..0000000
--- a/MooseX-StrictConstructor-0.12/lib/MooseX/StrictConstructor/Role/Meta/Method/Constructor.pm
+++ /dev/null
@@ -1,92 +0,0 @@
-package MooseX::StrictConstructor::Role::Meta::Method::Constructor;
-BEGIN {
-  $MooseX::StrictConstructor::Role::Meta::Method::Constructor::VERSION = '0.12';
-}
-
-use strict;
-use warnings;
-
-use B ();
-use Carp ();
-
-use Moose::Role;
-
-around '_generate_BUILDALL' => sub {
-    my $orig = shift;
-    my $self = shift;
-
-    my $source = $self->$orig();
-    $source .= ";\n" if $source;
-
-    my @attrs = (
-        '__INSTANCE__ => 1,',
-        map { B::perlstring($_) . ' => 1,' }
-        grep {defined}
-        map  { $_->init_arg() } @{ $self->_attributes() }
-    );
-
-    $source .= <<"EOF";
-my \%attrs = (@attrs);
-
-my \@bad = sort grep { ! \$attrs{\$_} }  keys \%{ \$params };
-
-if (\@bad) {
-    Carp::confess "Found unknown attribute(s) passed to the constructor: \@bad";
-}
-EOF
-
-    return $source;
-};
-
-no Moose::Role;
-
-1;
-
-# ABSTRACT: A role to make immutable constructors strict
-
-
-
-=pod
-
-=head1 NAME
-
-MooseX::StrictConstructor::Role::Meta::Method::Constructor - A role to make immutable constructors strict
-
-=head1 VERSION
-
-version 0.12
-
-=head1 SYNOPSIS
-
-  Moose::Util::MetaRole::apply_metaroles(
-      for_class => $caller,
-      class     => {
-          constructor =>
-              ['MooseX::StrictConstructor::Role::Meta::Method::Constructor'],
-      },
-  );
-
-=head1 DESCRIPTION
-
-This role simply wraps C<_generate_BUILDALL()> (from
-C<Moose::Meta::Method::Constructor>) so that immutable classes have a
-strict constructor.
-
-=head1 AUTHOR
-
-Dave Rolsky <autarch@urth.org>
-
-=head1 COPYRIGHT AND LICENSE
-
-This software is Copyright (c) 2010 by Dave Rolsky.
-
-This is free software, licensed under:
-
-  The Artistic License 2.0
-
-=cut
-
-
-__END__
-
-
diff --git a/MooseX-StrictConstructor-0.12/lib/MooseX/StrictConstructor.pm b/MooseX-StrictConstructor-0.12b/lib/MooseX/StrictConstructor.pm
index c5b28dc..968a7b8 100644
--- a/MooseX-StrictConstructor-0.12/lib/MooseX/StrictConstructor.pm
+++ b/MooseX-StrictConstructor-0.12b/lib/MooseX/StrictConstructor.pm
@@ -10,12 +10,12 @@ use Moose 0.94 ();
 use Moose::Exporter;
 use Moose::Util::MetaRole;
 use MooseX::StrictConstructor::Role::Object;
-use MooseX::StrictConstructor::Role::Meta::Method::Constructor;
+use MooseX::StrictConstructor::Role::Meta::Class;
 
 Moose::Exporter->setup_import_methods(
     class_metaroles => {
-        constructor =>
-            ['MooseX::StrictConstructor::Role::Meta::Method::Constructor']
+        class =>
+            ['MooseX::StrictConstructor::Role::Meta::Class']
     },
     base_class_roles => ['MooseX::StrictConstructor::Role::Object'],
 );
