diff -ur VCI-0.5.4.orig/lib/VCI/VCS/Cvs/File.pm VCI-0.5.4/lib/VCI/VCS/Cvs/File.pm
--- VCI-0.5.4.orig/lib/VCI/VCS/Cvs/File.pm	2010-08-13 08:49:06.000000000 -0400
+++ VCI-0.5.4/lib/VCI/VCS/Cvs/File.pm	2010-09-17 11:03:57.000000000 -0400
@@ -19,14 +19,18 @@
 sub _build_time {
     my $self = shift;
     my $rev = $self->revision;
+
+    # Force timezone to UTC in order to get consistent commit times
+    local $ENV{TZ} = 'UTC';
     my $output = $self->project->repository->vci->x_do(
         args => ['-n', 'log', '-N', "-r$rev", $self->name],
         fromdir => $self->parent->x_cvs_dir);
-    $output =~ /^date: (\S+ \S+);/ms;
+    $output =~ /^date: (\S+ \S+)( \S+)?;/ms;
     my $time = $1;
+    my $tz = $2 || 'UTC';
     confess("Failed to parse time for " . $self->path->stringify . " $rev")
         if !defined $time;
-    return "$time UTC";
+    return "$time $tz";
 }
 
 sub _build_content {
Only in VCI-0.5.4/lib/VCI/VCS/Cvs: File.pm~
Only in VCI-0.5.4/lib/VCI/VCS/Cvs: File.pm.orig
Only in VCI-0.5.4/lib/VCI/VCS/Cvs: File.pm.rej
