Index: spamd/spamd.raw
===================================================================
--- spamd/spamd.raw	(revision 559081)
+++ spamd/spamd.raw	(working copy)
@@ -1024,10 +1024,11 @@
       # use the POSIX functions to hide the platform specific workarounds 
       POSIX::setgid($ugid);  # set effective and real gid
       POSIX::setuid($uuid);  # set effective and real UID
+      $< = $uuid; $> = $uuid;   # bug 5574
 
       # keep the sanity check to catch problems like bug 3900 just in case
       if ( $> != $uuid and $> != ( $uuid - 2**32 ) ) {
-        die "spamd: setuid to uid $uuid failed\n";
+        die "spamd: setuid to uid $uuid failed (> = $>, < = $<)\n";
       }
     }
 
Index: lib/Mail/SpamAssassin/Util.pm
===================================================================
--- lib/Mail/SpamAssassin/Util.pm	(revision 559081)
+++ lib/Mail/SpamAssassin/Util.pm	(working copy)
@@ -1336,6 +1336,7 @@
     # bug 3586: kludges needed to work around platform dependent behavior assigning to $<
     #  The POSIX functions deal with that so just use it here
     POSIX::setuid($touid);
+    $< = $touid; $> = $touid;       # bug 5574
 
     # Check that we have now accomplished the setuid: catch bug 3586 if it comes back
     if ($< != $touid) {
